tools/make-rpms

```diff
@@ -25,15 +25,16 @@ base=$(cd $(dirname $0)/../; pwd -P)
 
 usage()
 {
-	echo "usage: make-rpms [--clean] [--quick] [--verbose]"
+	echo "usage: make-rpms [--clean] [--quick] [--verbose] [--tag T]"
```

Comment:
Could we call this a revision, as with virt-builder? Our git repository already has "tags", and we use them a lot in the continuous delivery system.